### PR TITLE
Add Calculator component

### PR DESCRIPTION
**Pull request by SFarida**
### This PR contains the following:
- components folder containing Calculator.js where two functional components are defined Table and Calculator. The Table component is a child to the Calculator.
- Calculator imported in the top level App component to be render

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+function Table() {
```
Comment (franclobo) [inline src/components/Calculator.js]: - [ ] Well done on creating a function to fill the page for good practice in REACT to having just one independent function on each file. Please 🙏🏽 divide it into two files one for Table.js and the other for calculator where you will add the corresponding function of the file.
- [ ] This is a REACT file please use it at the top of your document `import REACT from react;` for semantical reasons.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+export default function Calculator() {
+  return (
+    <div className="container py-5">
+      <div className="table-responsive container">
+        <Table />
+      </div>
+    </div>
+  );
+}
```
Comment (franclobo) [inline src/components/Calculator.js:12]: - [ ] - [ ] Well done on creating a function to fill the page for good practice in REACT to having just one independent function on each file. Please 🙏🏽 divide it into two files one for Table.js and the other for calculator where you will add the corresponding function of the file.
- [ ] This is a REACT file please use it at the top of your document `import REACT from react;` for semantical reasons and in Calculator.js file after import REACT add `import Table from './Table';`

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <tr className="numbers">
+          <td colSpan={2} className="py-3">0</td>
+          <td className="py-3">.</td>
+          <td className="py-3 operator">=</td>
+        </tr>
+      </tbody>
+    </table>
+  );
+}
+
```
Comment (franclobo) [inline src/components/Calculator.js:3]: - [ ] Here you can add `export default Table;` to export the function Table and be able to use it in the Calculator component.

File: src/index.css [hunk 4]
```diff
@@ -16,12 +16,15 @@ body {
...
+  background-color: #848693;
+  text-align: end;
+}
+
+.operator {
+  background-color: #f39148 !important;
+}
+
+.numbers {
+  background-color: #e0e0e0;
```
Comment (franclobo) [inline src/index.css:29]: - [ ] Please correct the dimensions of the Calculator to look similar to the given design, this is a desktop web page 👇🏽 

| Original design | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @SFarida, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Correct Git flow ✔️ 
- Components are functional ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] For a professional README file please complete all the main sections according with the current project 👇🏽 
    - Kindly add a description
image
   - Kindly list the tech that you used to code your program.
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 5]
```diff
@@ -157,8 +157,8 @@ Not yet deployed
 
 > Describe 1 - 3 features you will add to the project.
 
-<!-- - [ ] **Displays scores**
-- [ ] **Submit scores** -->
+- [ ] **Make calculations**
+<!-- - [ ] **Submit scores** -->
```
Comment (BenMukebo) [inline README.md:162]: - [OPTIONAL] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

**Review by BenMukebo — APPROVED** [review]
### Hi @SFarida, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- The `Calculator.js` component is a **functional** component:heav_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] All the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. But  there is still some boilerplate files. If you heck your src directory you can see these file `App.test.js, reportWebVitals.js, setupTests.js, logo.svg` and you still have some icons and images in the public folder that need to be deleted. Kindly remove it 😉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**